doc-content/enterprise-only/bpmn/bpmn-gateways-ref.adoc

```diff
@@ -13,9 +13,13 @@ image:BPMN2/gateway.png[]
 In this example, the customer details are verified by a user and the process is assigned to a user for approval. If approved, an approval notification is sent to the user. If the event of the request is rejected, a rejection notification is sent to the user.
 
 
-.Gateway elements table
-[cols="2"]
+.Gateway elements
+
+[cols="50%,50%" options="header"]
+
 |===
+h|Element type
+h|
```

Comment:
Ditto, from my comment for "End events". Second column should also be headed, even if just "Icon".